docs/source/index.rst

```diff
@@ -45,7 +46,30 @@ Our documentation is organized into a few sections:
    getting_started/using_composer.rst
    getting_started/welcome_tour.rst
    tutorials/adding_models_datasets.rst
-   getting_started/distributed.rst
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Using our Trainer
+
+   trainer/examples.rst
+   trainer/mosaic_model.rst
+   trainer/algorithms.rst
+   trainer/checkpointing.rst
```

Comment:
Let's put everything checkpointing and below in a separate topic, e.g. "Trainer details" or something similar.